classes/class-wc-connect-shipping-label.php

```diff
@@ -111,22 +117,38 @@ protected function get_selected_packages(WC_Order $order ) {
 		}
 
 		protected function get_all_packages() {
-			$packages = $this->settings_store->get_packages();
-
-			if ( ! $packages ) {
-				return new stdClass();
-			}
+			$custom_packages = $this->settings_store->get_packages();
 
 			$formatted_packages = array();
 
-			foreach( $packages as $package ) {
+			foreach( $custom_packages as $package ) {
 				$package_id = $package[ 'name' ];
 				$formatted_packages[ $package_id ] = $package;
 			}
 
+			$flat_rate_schema = $this->service_schemas_store->get_predefined_packages_schema_for_service( 'usps' );
+
+			foreach ( $flat_rate_schema as $group ) {
+				foreach ( $group->definitions as $package ) {
+					$package_id = property_exists( $package, 'id' ) ? $package->id : $package->name;
+					$formatted_packages[ $package_id ] = $package;
+				}
+			}
+
 			return $formatted_packages;
 		}
 
+		protected function get_flat_rate_packages_groups() {
+			$flat_rate_schema = $this->service_schemas_store->get_predefined_packages_schema_for_service( 'usps' );
```

Comment:
We will be adding non flat rate predefined packages next (like these: https://store.usps.com/store/browse/uspsProductDetailMultiSkuDropDown.jsp?categoryNavIds=shipping-supplies%3afree-shipping-supplies&categoryNav=false&navAction=push&navCount=0&atg.multisite.remap=false&categoryId=free-shipping-supplies&productId=P_O_BOX4 )

Please filter the packages returned by `get_predefined_packages_schema_for_service ` for flat rate packages only or a TODO comment here to fix this when we add non flat rate predefined packages.